core/arch/arm/sm/sm.c

```diff
@@ -39,6 +39,11 @@ bool sm_from_nsec(struct sm_ctx *ctx)
 {
 	uint32_t *nsec_r0 = (uint32_t *)(&ctx->nsec.r0);
 
+#ifdef CFG_SM_PLATFORM_HANDLER
```

Comment:
This ifdef isn't needed as `sm_platform_handler()` always returns `true` (as a static inline function) when `CFG_SM_PLATFORM_HANDLER` isn't defined.